### ISSUE TITLE
ntfy web crash when choose a language with underline in name

### ISSUE THREAD

**Issue by Potat0000** [binwiederhier/ntfy#841]
:lady_beetle: **Describe the bug**
With the latest v2.7.0, I found that ntfy web will crash when choose a language with underline in name, like `zh_Hans (中文)` and `pt_BR (Português (Brasil))`, but will act normal with a language without underline.

:computer: **Components impacted**
Web APP with v2.7.0

:bulb: **Screenshots and/or logs**
![image](https://github.com/binwiederhier/ntfy/assets/11223244/25ad745b-37f5-4d71-8718-ba44cb497610)
```
RangeError: Incorrect locale information provided
  at log (https://ntfy.domain.tld/static/media/index-9e24212a.js:320:286)
  at lse (https://ntfy.domain.tld/static/media/index-9e24212a.js:281:326)
  at isMounted (https://ntfy.domain.tld/static/media/index-9e24212a.js:19518:38)
  at callback (https://ntfy.domain.tld/static/media/index-9e24212a.js:44027:40)
  at callback (https://ntfy.domain.tld/static/media/index-9e24212a.js:39763:40)
  at callback (https://ntfy.domain.tld/static/media/index-9e24212a.js:39691:40)
  at callback (https://ntfy.domain.tld/static/media/index-9e24212a.js:39545:40)
  at callback (https://ntfy.domain.tld/static/media/index-9e24212a.js:35913:40)
  at callback (https://ntfy.domain.tld/static/media/index-9e24212a.js:34864:40)

RangeError: Incorrect locale information provided
  at at use (https://ntfy.domain.tld/static/media/index-9e24212a.js:326:238)
  at     at div
  at     at https://ntfy.domain.tld/static/media/index-9e24212a.js:55:6185
  at     at https://ntfy.domain.tld/static/media/index-9e24212a.js:61:22350
  at     at div
  at     at https://ntfy.domain.tld/static/media/index-9e24212a.js:55:6185
  at     at https://ntfy.domain.tld/static/media/index-9e24212a.js:61:20669
  at     at div
  at     at div
  at     at t (https://ntfy.domain.tld/static/media/index-9e24212a.js:263:1464)
  at     at IR (https://ntfy.domain.tld/static/media/index-9e24212a.js:286:86948)
  at     at ise (https://ntfy.domain.tld/static/media/index-9e24212a.js:286:86525)
  at     at rse (https://ntfy.domain.tld/static/media/index-9e24212a.js:286:86333)
  at     at oK (https://ntfy.domain.tld/static/media/index-9e24212a.js:240:3933)
  at     at pK (https://ntfy.domain.tld/static/media/index-9e24212a.js:240:6226)
  at     at main
  at     at https://ntfy.domain.tld/static/media/index-9e24212a.js:55:6185
  at     at https://ntfy.domain.tld/static/media/index-9e24212a.js:61:13098
  at     at Rde (https://ntfy.domain.tld/static/media/index-9e24212a.js:409:15094)
  at     at div
  at     at https://ntfy.domain.tld/static/media/index-9e24212a.js:55:6185
  at     at https://ntfy.domain.tld/static/media/index-9e24212a.js:61:13098
  at     at Ode (https://ntfy.domain.tld/static/media/index-9e24212a.js:409:13968)
  at     at oK (https://ntfy.domain.tld/static/media/index-9e24212a.js:240:3933)
  at     at gK (https://ntfy.domain.tld/static/media/index-9e24212a.js:240:6851)
  at     at Cce (https://ntfy.domain.tld/static/media/index-9e24212a.js:372:5834)
  at     at i (https://ntfy.domain.tld/static/media/index-9e24212a.js:250:3140)
  at     at GN (https://ntfy.domain.tld/static/media/index-9e24212a.js:61:18729)
  at     at YN (https://ntfy.domain.tld/static/media/index-9e24212a.js:61:19114)
  at     at D3 (https://ntfy.domain.tld/static/media/index-9e24212a.js:61:29727)
  at     at fK (https://ntfy.domain.tld/static/media/index-9e24212a.js:240:6285)
  at     at _K (https://ntfy.domain.tld/static/media/index-9e24212a.js:249:744)
  at     at Pde (https://ntfy.domain.tld/static/media/index-9e24212a.js:409:12627)
  at     at Suspense
  at     at Ade (https://ntfy.domain.tld/static/media/index-9e24212a.js:409:12872)
```

**Comment by onepiecewyz** [comment]
I encountered the same bug.

**Comment by binwiederhier** [comment]
Oh dear. That's unfortunate.

**Comment by binwiederhier** [comment]
What browser are you using? I tried this on Chrome+Firefox without "success". It doesn't crash. Can you elaborate on your steps?

**Comment by Potat0000** [comment]
> What browser are you using? I tried this on Chrome+Firefox without "success". It doesn't crash. Can you elaborate on your steps?

I'm using Edge 116.0.1938.69.

There isn't any special step to reproduce it. Just open web app and login. If it's already in language with underline, it will crash immediately. Or you could change language after login and it will crash with choose a subscribed topics.

**Comment by YamatoRyou** [comment]
Same.
ntfy 2.7.0
Chromium 102.0.5005.167

**Comment by Potat0000** [comment]
fixed in #922
